src/app/models/profile.ts

```diff
@@ -6,8 +6,11 @@ export class Profile {
     bio?: string;
     username?: string;
     url?: string;
-    emails?: Email[];
-    primaryEmail?: Email;
+    emails?: string[];
```

Comment:
Does it really make sense to enumerate the emails like this, instead of having email and email type? This doesn't scale as nicely.